include/dp_vnf.h

```diff
@@ -17,6 +17,14 @@ extern "C" {
 
 #define DP_VNF_MATCH_ALL_PORT_IDS 0xFFFF
 
+#define DP_PRINT_VNF_OP_WARNING(MESSAGE, VNF) \
+		if (VNF->alias_pfx.ol.ip_type == RTE_ETHER_TYPE_IPV4) {\
+			DPS_LOG_WARNING(MESSAGE, DP_LOG_VNF_TYPE(VNF->type), DP_LOG_VNI(VNF->vni), \
+							DP_LOG_PORTID(VNF->port_id), DP_LOG_IPV4(VNF->alias_pfx.ol.ipv4), DP_LOG_PREFLEN(VNF->alias_pfx.length)); } \
+		else {\
+			DPS_LOG_WARNING(MESSAGE, DP_LOG_VNF_TYPE(VNF->type), DP_LOG_VNI(VNF->vni), \
+							DP_LOG_PORTID(VNF->port_id), DP_LOG_IPV6(VNF->alias_pfx.ol.ipv6), DP_LOG_PREFLEN(VNF->alias_pfx.length)); } \
+
```

Comment:
I like the encapsulation for more informative logs.
But, unless you aim to use this macro from another module, it would be best to keep in in the `.c` file. It also can be an inline instead of a macro.
And the name would better be `DP_LOG_VNF_WARNING`, as `PRINT` evokes `printf()`. Not sure what the `_OP_` means, I guess a typo of IP? But even then it is used even for non-IP related warnings, so that's why I would remove that infix also.

Reply:
I agree with the critics on including PRINT. The OP means operation, not a typo.